sources/styles.css

```diff
@@ -147,3 +143,296 @@ div.button i.fa-arrow-down {
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button3 {
+  display: flex;
+  width: 82px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button4 {
+  display: flex;
+  width: 51px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+section.main-button {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  align-self: center;
+}
+
+button.projectview {
+  background-color: #36b37f;
+  color: #ffff;
+  margin-top: 29px;
+  margin-bottom: 24px;
+  width: 122px;
+  height: 48px;
+  border-radius: 4px;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
+
+button.projectview:hover {
+  background-color: #3f9973;
+}
+
+button.projectview:active {
+  background-color: #008552;
+  box-shadow: 0 5px rgb(47, 46, 46);
+}
+
+/* about me */
+section.about {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  box-sizing: border-box;
+  margin-top: 18px;
+  width: 100%;
+  justify-content: center;
+}
+
+div.aboutme-desc h2 {
+  display: flex;
+  justify-content: center;
+  width: 351px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-bottom: 24px;
+}
+
+div.aboutme-desc p {
+  display: flex;
+  text-align: center;
+  width: 351px;
+  height: 189px;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 32px;
+  color: #172b4d;
+}
+
+section.resume-btn {
+  display: flex;
+  justify-content: center;
+}
+
+section.resume-btn button {
+  background-color: #36b37f;
+  color: #ffff;
+  margin-top: 29px;
+  margin-bottom: 24px;
+  width: 122px;
+  height: 48px;
+  border-radius: 4px;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
```

Comment:
- [ ] Kindly make the necessary changes so your button better matches the design, like the padding and the border
Yours:
![image](https://user-images.githubusercontent.com/73140461/148589523-b0232b9d-0ce7-4d7d-b91f-c26bc1abb94f.png)
Design:
![image](https://user-images.githubusercontent.com/73140461/148589806-e8388118-dd62-4c66-920f-ace7d16ba9a7.png)